client/post-editor/editor-featured-image/index.jsx

```diff
@@ -26,12 +27,14 @@ import { recordTracksEvent } from 'state/analytics/actions';
 
 class EditorFeaturedImage extends Component {
 	static propTypes = {
-		maxWidth: React.PropTypes.number,
-		site: React.PropTypes.object,
-		post: React.PropTypes.object,
-		selecting: React.PropTypes.bool,
-		onImageSelected: React.PropTypes.func,
-		featuredImage: React.PropTypes.object,
+		maxWidth: PropTypes.number,
+		site: PropTypes.object,
+		post: PropTypes.object,
+		selecting: PropTypes.bool,
+		onImageSelected: PropTypes.func,
+		featuredImage: PropTypes.object,
+		hasDropZone: PropTypes.bool,
```

Comment:
I don't see `hasDropZone` and `isDropZoneVisible` inside this file. Is this coming from other PR? :)

Reply:
Ah, you're right, I'll remove those, good catch :)